src/core_plugins/kibana/public/context/api/utils/sorting.js

```diff
@@ -1,5 +1,33 @@
 import _ from 'lodash';
 
+
+/**
+ * The list of field names that are allowed for sorting, but not included in
+ * index pattern fields.
+ *
+ * @constant
+ * @type {string[]}
+ */
+const META_FIELD_NAMES = ['_seq_no', '_doc', '_uid'];
+
+/**
+ * Returns a field from the intersection of the set of sortable fields in the
+ * given index pattern and a given set of candidate field names.
+ *
+ * @param {IndexPattern} indexPattern - The index pattern to search for
+ *     sortable fields
+ * @param {string[]} fields - The list of candidate field names
+ *
+ * @returns {string[]}
+ */
+function getFirstSortableField(indexPattern, fieldNames) {
+  const sortableFields = fieldNames.filter((fieldName) => (
+    META_FIELD_NAMES.includes(fieldName)
+    || (indexPattern.fields.byName[fieldName] || { sortable: false }).sortable
```

Comment:
In this case we're sending an invalid query to elasticsearch though, which I don't think we should do in the first place. The `sort` portion of the query looks like this:

```
  "sort": [
    {
      "@timestamp": {
        "order": "desc",
        "unmapped_type": "boolean"
      }
    },
    {
      "undefined": {
        "order": "asc",
        "unmapped_type": "boolean"
      }
    }
  ],
```

Instead, what if we fail early if there are no valid tiebreaker fields and notify the user with a message saying they need to set a valid tiebreaker field?